packages/utils/dist/index.es.js

```diff
@@ -1,4 +1,5 @@
 import { AtomicBlockUtils, RichUtils, EditorState, Modifier } from 'draft-js';
+import punycode from 'punycode';
```

Comment:
How big is this dependency? Also, it needs to be added to `external` in the rollup config

Reply:
why do we need this exactly?

Reply:
This dependency used for handle unicode characters. 

It's size is 142B(MINIFIED). Still need to added it into `external`?